lldb/test/API/functionalities/data-formatter/data-formatter-stl/libstdcpp/map/TestDataFormatterStdMap.py

```diff
@@ -19,9 +23,22 @@ def setUp(self):
     @add_test_categories(["libstdcxx"])
     @expectedFailureAll(bugnumber="llvm.org/pr50861", compiler="gcc")
     def test_with_run_command(self):
+        build_args = {"EXE": "a.out"}
+        self.with_run_command("", build_args)
+
+    @add_test_categories(["libstdcxx"])
+    @expectedFailureAll(bugnumber="llvm.org/pr50861", compiler="gcc")
+    def test_with_run_command_debug(self):
+        build_args = {"CXXFLAGS": "-D_GLIBCXX_DEBUG", "EXE": "debug_a.out"}
```

Comment:
Why does the `EXE` need to be different here? I think setting `CXXFLAGS` should be sufficient?